optimum/habana/distributed/strategy.py

```diff
@@ -0,0 +1,112 @@
+from abc import abstractmethod
+from typing import Any, List, Mapping
+
+import torch
+import torch.distributed
+from torch import nn
+
+#from optimum.habana.distributed import tp_wrapping
```

Comment:
minor: please  remove the commented code

Reply:
> minor: please remove the commented code

Done